homework_4/bet_acceptance_api/internal/domain/services/betService.go

```diff
@@ -0,0 +1,19 @@
+package services
+
+import (
+	"github.com/superbet-group/code-cadets-2021/homework_4/bet_accceptance_api/internal/api/controllers/models"
+)
+
+type BetService struct {
+	betPublisher BetPublisher
+}
+
+func NewBetService(betPublisher BetPublisher) *BetService {
+	return &BetService{
+		betPublisher: betPublisher,
+	}
+}
+
+func (b *BetService) CreateBet(bet models.BetRequestDto) error {
```

Comment:
service should not know about the controller models, it should work with domain models or just accept ticket creation related parameters `CreateBet(payment, coefficient,....)`